feed.rss

```diff
@@ -255,5 +255,57 @@
 			]]>
 		</itunes:summary>
 	</item>
+	<item>
+		<guid>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_5/The.Context.episode.5.mp3</guid>
+		<title>Episode 5: Android TV with Joe Birch</title>
+		<link>https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_5/The.Context.episode.5.mp3</link>
+		<itunes:duration>49:22</itunes:duration>
+		<pubDate>Fri, 1 June 2016 19:06:00 EST</pubDate>
+		<enclosure url="https://github.com/artem-zinnatullin/TheContext-Podcast/releases/download/Episode_5/The.Context.episode.5.mp3" length="47396669" type="audio/mpeg"/>
```

Comment:
Of course we will share the cost of hosting ... I guess we can try it for a Month, what do you think? Alternative there is a german hoster called uberspace. Basically they offer a pay as much as you want service (min is 1 Euro per month) with 10 GB disk space and unlimited traffic https://uberspace.de/prices#cost (maybe google translate could do a decent job here)